### ISSUE TITLE
Bug Report

### ISSUE THREAD

**Issue by juanicarcamo** [OpenTTD/OpenTTD#9125]
## Version of OpenTTD

OpenTTD version:
 Version:    1.11.1 (0)
 NewGRF ver: 1b086d64
 Bits:       64
 Endian:     little
 Dedicated:  no
 Build date: Apr 18 2021 21:16:51
## Actual result

Crash reason:
 Exception: C0000005
 Location:  00007FF7BE3A3820
 Message:   <none>

## Steps to reproduce

[crash.log](https://github.com/OpenTTD/OpenTTD/files/6388037/crash.log)

```
*** OpenTTD Crash Report ***

Crash at: Tue Apr 27 22:37:35 2021
In game date: 0-01-01 (0)

Crash reason:
 Exception: C0000005
 Location:  00007FF7BE3A3820
 Message:   <none>

OpenTTD version:
 Version:    1.11.1 (0)
 NewGRF ver: 1b086d64
 Bits:       64
 Endian:     little
 Dedicated:  no
 Build date: Apr 18 2021 21:16:51

Registers:
 RAX: 0000000000000001 RBX: 000000000000023F RCX: 00000000FF737573 RDX: 0000000000000000
 RSI: 000001C246C80C00 RDI: 0000000000000001 RBP: 0000000000000007 RSP: 0000007CD21CF230
 R8:  000001C24DCB2040 R9:  0000000000008007 R10: 0000000000000000 R11: 000001C24DCB2040
 R12: 0000000000000000 R13: 0000000000000000 R14: 0000000000008000 R15: 000000000000023F
 RIP: 00007FF7BE3A3820 EFLAGS: 00010282

 Bytes at instruction pointer:
 89 0A 48 8D 52 04 FF C8 66 45 89 08 4D 8D 40 02 85 C0 7F EC 44 8B 0D 9D

Stack trace:
 00000000 00000000 00000020 00000000 00000000 00000000 69B6C27C 00000000
 48ABDF01 000001C2 4A7B6180 000001C2 46C80C00 000001C2 BE56C563 00007FF7
 00000000 00000000 00000001 00000000 D21CF540 0000007C 00000000 00000000
 4A7B6107 000001C2 4A7B6180 000001C2 0000023F 00000000 000003FF 00000000
 00000000 00000000 00000001 00000000 00000161 00000000 00000000 00000000
 00000000 00000000 BE746434 00007FF7 00000000 00000000 00000000 000001C2
 00000000 00000000 00000000 00000000 00000007 000001C2 00000000 00000000
 00000005 00000003 00000007 000001C2 00000000 00000000 00000000 00000000
 4DE240F0 000001C2 0000029F 00000000 46D0FA70 000001C2 BE749E07 00007FF7
 4DE240F0 000001C2 00000161 00000000 00000240 00000000 691E7640 00000000
 00000006 00000000 00000000 00000000 00000000 00000000 48ABDF10 000001C2
 00000000 00000000 000003FF 0000023F 2C8165B7 0000D2B2 1F0725A1 00007FFE
 4DE240F0 000001C2 BE74831C 00007FF7 46CE9750 000001C2 D21CF508 0000007C
 0000029F 00000000 46D0FA70 000001C2 000000EA 00000000 BE747619 00007FF7
 D21CF540 0000007C 00000000 00000000 48ADD7D0 000001C2 69B7CB85 00000000
 D21CF4A0 0000007C FEFB2253 00007FFD FFFFFFFF FFFFFFFF 6A7FD080 00000000
 4A447450 000001C2 69B6153D 00000000 00000156 00000000 BE74E27C 00007FF7
 B32B201E 0000F2B2 46BA0000 000001C2 FFFFFFFF FFFFFFFF BE8DFE10 00007FF7
 00000001 00000000 6904CC4C 00000000 00000000 00000000 00000000 00000000
 00000240 00000000 00000400 00000000 000000EA 00000000 BE74E19A 00007FF7
 4DE240F0 000001C2 00000161 00000000 00000156 00000000 0000029F 00000000
 00000240 000001C2 46C9EC60 000001C2 BEA213C0 00007FF7 00000000 00000000
 00000400 00000000 00000240 00000000 00000400 00000000 BE74E34D 00007FF7
 4DE240F0 000001C2 4DE240F0 000001C2 00000000 00000000 00000240 00000000

Operating system:
 Name:     Windows
 Release:  10.0.19042 ()
 Compiler: MSVC 1928

Configuration:
 Blitter:      32bpp-anim
 Graphics set: none (4294967295)
 Language:     C:\Users\JUANIG~1\AppData\Local\Temp\Rar$EXa6096.37913\openttd-1.11.1-windows-win64\lang\spanish.lng
 Music driver: none
 Music set:    none (4294967295)
 Network:      no
 Sound driver: none
 Sound set:    none (4294967295)
 Video driver: win32-opengl

Fonts:
 Small:  Arial
 Medium: Arial
 Large:  Arial
 Mono:   sprite

AI Configuration (local: 0) (current: 0):

Libraries:
 LZMA:       5.2.5
 LZO:        2.10
 PNG:        1.6.37
 Zlib:       1.2.11

Module information:
 C:\Users\JUANIG~1\AppData\Local\Temp\Rar$EXa6096.37913\openttd-1.11.1-windows-win64\openttd.exe handle: 00007FF7BE260000 size: 8553984 crc: 9A796EBF date: 2021-04-18 21:22:06
 C:\WINDOWS\SYSTEM32\ntdll.dll handle: 00007FFE21190000 size: 2024728 crc: 4399D735 date: 2021-04-21 17:49:36
 C:\Program Files\AVG\Antivirus\aswhook.dll handle: 00007FFDFEFB0000 size: 64056 crc: C833D4F2 date: 2021-04-09 14:17:53
 C:\WINDOWS\System32\KERNEL32.DLL handle: 00007FFE1FD30000 size: 764976 crc: 3A191151 date: 2021-04-21 17:49:03
 C:\WINDOWS\System32\KERNELBASE.dll handle: 00007FFE1ED40000 size: 2919288 crc: EAF99E19 date: 2021-04-21 17:49:39
 C:\WINDOWS\System32\WS2_32.dll handle: 00007FFE20220000 size: 427200 crc: 6E37A637 date: 2021-03-23 10:21:51
 C:\WINDOWS\System32\RPCRT4.dll handle: 00007FFE1F4C0000 size: 1220520 crc: BB501898 date: 2021-04-21 17:49:39
 C:\WINDOWS\System32\IMM32.dll handle: 00007FFE201F0000 size: 185448 crc: 36F61BF6 date: 2021-03-23 10:21:53
 C:\WINDOWS\System32\USER32.dll handle: 00007FFE1F970000 size: 1702416 crc: 73C75F75 date: 2021-04-21 17:49:18
 C:\WINDOWS\System32\win32u.dll handle: 00007FFE1ED10000 size: 132760 crc: 167E1EAD date: 2021-04-21 17:49:18
 C:\WINDOWS\System32\GDI32.dll handle: 00007FFE1F940000 size: 161680 crc: 2A141BAA date: 2021-03-23 10:20:33
 C:\WINDOWS\System32\gdi32full.dll handle: 00007FFE1F010000 size: 1075880 crc: 6C19849E date: 2021-04-21 17:49:49
 C:\WINDOWS\System32\msvcp_win.dll handle: 00007FFE1E8D0000 size: 634760 crc: E6732639 date: 2021-03-23 10:21:45
 C:\WINDOWS\System32\ucrtbase.dll handle: 00007FFE1EAB0000 size: 1044888 crc: D3CA62A3 date: 2021-03-23 10:21:45
 C:\WINDOWS\System32\SHELL32.dll handle: 00007FFE20970000 size: 7637008 crc: 57C77621 date: 2021-04-21 17:49:51
 C:\WINDOWS\System32\ole32.dll handle: 00007FFE1F5F0000 size: 1212232 crc: 0FD6B05F date: 2021-03-23 10:21:43
 C:\WINDOWS\System32\combase.dll handle: 00007FFE1FDF0000 size: 3506992 crc: C1B1E60F date: 2021-04-21 17:49:34
 C:\WINDOWS\System32\ADVAPI32.dll handle: 00007FFE1F350000 size: 689024 crc: 355A9B19 date: 2021-03-23 10:20:03
 C:\WINDOWS\System32\msvcrt.dll handle: 00007FFE1F8A0000 size: 637360 crc: 072737D9 date: 2021-03-23 10:21:49
 C:\WINDOWS\System32\sechost.dll handle: 00007FFE20150000 size: 625496 crc: E09B47D4 date: 2021-04-21 17:49:39
 C:\WINDOWS\SYSTEM32\OPENGL32.dll handle: 00007FFDDD200000 size: 1064448 crc: A63281AE date: 2021-04-21 17:50:25
 C:\WINDOWS\SYSTEM32\WINMM.dll handle: 00007FFE05F80000 size: 147200 crc: 18D40339 date: 2021-03-23 10:19:57
 C:\WINDOWS\SYSTEM32\USP10.dll handle: 00007FFE097A0000 size: 79360 crc: 9A7F7CC2 date: 2021-03-23 10:21:53
 C:\WINDOWS\SYSTEM32\GLU32.dll handle: 00007FFE0D510000 size: 164352 crc: DA0CBD61 date: 2021-03-23 10:24:44
 C:\WINDOWS\SYSTEM32\windows.storage.dll handle: 00007FFE1CA10000 size: 7968600 crc: A4E92974 date: 2021-04-21 17:49:17
 C:\WINDOWS\SYSTEM32\Wldp.dll handle: 00007FFE1E340000 size: 173016 crc: C5218D0B date: 2021-03-23 10:21:45
 C:\WINDOWS\System32\SHCORE.dll handle: 00007FFE1F1F0000 size: 704008 crc: 0812F61E date: 2021-03-23 10:21:22
 C:\WINDOWS\System32\shlwapi.dll handle: 00007FFE1F400000 size: 342440 crc: 5693748E date: 2021-03-23 10:22:12
 C:\WINDOWS\system32\uxtheme.dll handle: 00007FFE1C330000 size: 627712 crc: A05B5F55 date: 2021-03-23 10:22:04
 C:\WINDOWS\System32\MSCTF.dll handle: 00007FFE1F720000 size: 1128008 crc: 3E23452A date: 2021-04-21 17:49:47
 C:\WINDOWS\System32\OLEAUT32.dll handle: 00007FFE20310000 size: 831024 crc: 8304FB12 date: 2021-03-23 10:21:48
 C:\WINDOWS\SYSTEM32\kernel.appcore.dll handle: 00007FFE1C810000 size: 60464 crc: 7039ACC9 date: 2021-03-23 10:20:57
 C:\WINDOWS\System32\bcryptPrimitives.dll handle: 00007FFE1F170000 size: 523200 crc: 1A4DFB8D date: 2021-03-23 10:21:51
 C:\WINDOWS\System32\clbcatq.dll handle: 00007FFE1F2A0000 size: 672976 crc: AD4CB4A3 date: 2021-03-23 10:21:43
 C:\WINDOWS\SYSTEM32\atig6pxx.dll handle: 00007FFE1A460000 size: 17920 crc: B6E883A9 date: 2015-01-13 19:20:52
 C:\WINDOWS\SYSTEM32\atio6axx.dll handle: 0000000069030000 size: 26017280 crc: 699A202A date: 2015-01-13 19:46:26
 C:\WINDOWS\System32\SETUPAPI.dll handle: 00007FFE203E0000 size: 4650576 crc: F65AC9E1 date: 2021-03-23 10:22:08
 C:\WINDOWS\System32\cfgmgr32.dll handle: 00007FFE1F120000 size: 305272 crc: DE2280BE date: 2021-03-23 10:21:48
 C:\WINDOWS\SYSTEM32\VERSION.dll handle: 00007FFE16750000 size: 31496 crc: 546E295F date: 2021-03-23 10:22:12
 C:\WINDOWS\System32\bcrypt.dll handle: 00007FFE1E9D0000 size: 146240 crc: 50F3C5E2 date: 2021-03-23 10:21:45
 C:\WINDOWS\SYSTEM32\dwmapi.dll handle: 00007FFE1C520000 size: 168408 crc: BE283206 date: 2021-03-23 10:21:43
 C:\WINDOWS\SYSTEM32\atiadlxx.dll handle: 00007FFE14930000 size: 535552 crc: A22303DE date: 2015-01-13 19:20:58
 C:\WINDOWS\System32\PSAPI.DLL handle: 00007FFE20850000 size: 19144 crc: 6E06CEEB date: 2021-03-23 10:21:42
 C:\WINDOWS\SYSTEM32\USERENV.dll handle: 00007FFE1E7D0000 size: 176656 crc: 82C8FFDF date: 2021-03-23 10:21:48
 C:\WINDOWS\SYSTEM32\WTSAPI32.dll handle: 00007FFE1A620000 size: 67192 crc: 7604358D date: 2021-03-23 10:20:08
 C:\WINDOWS\SYSTEM32\WSOCK32.dll handle: 00007FFE148F0000 size: 18944 crc: F78C7419 date: 2019-12-07 09:08:41
 C:\WINDOWS\SYSTEM32\DEVOBJ.dll handle: 00007FFE1E680000 size: 165832 crc: 9AB2CB9F date: 2021-03-23 10:21:48
 C:\WINDOWS\System32\WINTRUST.dll handle: 00007FFE1E970000 size: 382720 crc: 255FF0BB date: 2021-03-23 10:20:58
 C:\WINDOWS\System32\CRYPT32.dll handle: 00007FFE1EBB0000 size: 1425440 crc: 71673E8A date: 2021-03-23 10:21:46
 C:\WINDOWS\SYSTEM32\MSASN1.dll handle: 00007FFE1E4C0000 size: 64072 crc: FDFACF86 date: 2021-03-23 10:21:48
 C:\WINDOWS\SYSTEM32\aticfx64.dll handle: 00007FFE16780000 size: 1094024 crc: DAF97EC9 date: 2015-01-13 20:45:56
 C:\WINDOWS\SYSTEM32\atig6txx.dll handle: 00007FFE188F0000 size: 41984 crc: FEBD1D47 date: 2015-01-13 19:20:50
 C:\WINDOWS\SYSTEM32\textinputframework.dll handle: 00007FFE0C890000 size: 1016456 crc: 1AA07241 date: 2021-04-21 17:49:15
 C:\WINDOWS\SYSTEM32\CoreMessaging.dll handle: 00007FFE1BE70000 size: 986464 crc: EDF107E3 date: 2021-03-23 10:21:44
 C:\WINDOWS\SYSTEM32\CoreUIComponents.dll handle: 00007FFE1B790000 size: 3537520 crc: 78335519 date: 2021-03-23 10:20:32
 C:\WINDOWS\SYSTEM32\wintypes.dll handle: 00007FFE1A670000 size: 1394016 crc: BD196440 date: 2021-04-21 17:49:35
 C:\WINDOWS\SYSTEM32\ntmarta.dll handle: 00007FFE1D650000 size: 191656 crc: B2A51DA2 date: 2021-03-23 10:21:48
 C:\WINDOWS\SYSTEM32\dxgi.dll handle: 00007FFE1D210000 size: 979792 crc: 3FEC10DC date: 2021-04-21 17:49:14
 C:\WINDOWS\SYSTEM32\d3d11.dll handle: 00007FFE1AC30000 size: 2503504 crc: D4639E70 date: 2021-03-23 10:20:33
 C:\WINDOWS\SYSTEM32\atiuxp64.dll handle: 00007FFE16760000 size: 61464 crc: 1949A53C date: 2015-01-13 20:46:36
 C:\WINDOWS\SYSTEM32\apphelp.dll handle: 00007FFE1C190000 size: 570880 crc: EC8E5911 date: 2021-04-21 17:49:32
 C:\WINDOWS\SYSTEM32\AcGenral.DLL handle: 00007FFE15FB0000 size: 370176 crc: 4320519B date: 2021-03-23 10:24:50
 C:\WINDOWS\SYSTEM32\MPR.dll handle: 00007FFE15F90000 size: 109504 crc: 92B7C454 date: 2021-03-23 10:21:51
 C:\WINDOWS\SYSTEM32\SspiCli.dll handle: 00007FFE1E790000 size: 186976 crc: 6DAD5F72 date: 2021-04-21 17:49:38
 C:\WINDOWS\SYSTEM32\atidxx64.dll handle: 00007FFE16010000 size: 7569128 crc: 98A43701 date: 2015-01-13 20:45:48
 C:\WINDOWS\SYSTEM32\dxcore.dll handle: 00007FFE16890000 size: 219016 crc: 72966353 date: 2021-03-23 10:20:21
 C:\WINDOWS\system32\mswsock.dll handle: 00007FFE1E0B0000 size: 418416 crc: 7609CC3B date: 2021-03-23 10:21:45
 C:\WINDOWS\SYSTEM32\TextShaping.dll handle: 00007FFE0A7A0000 size: 707016 crc: 49A4E545 date: 2021-03-23 10:20:33

---- gamelog start ----
---- gamelog end ----

Recent news messages:

*** End of OpenTTD Crash Report ***

Decoded stack trace:
[00] openttd              0x00007FF7BE3A3820
[01] openttd              0x00007FF7BE56C563
[02] openttd              0x00007FF7BE746434
[03] openttd              0x00007FF7BE749E07
[04] openttd              0x00007FF7BE74831C
[05] openttd              0x00007FF7BE747619
[06] openttd              0x00007FF7BE74E27C
[07] openttd              0x00007FF7BE74E19A
[08] openttd              0x00007FF7BE74E34D
[09] openttd              0x00007FF7BE56EFC7
[10] openttd              0x00007FF7BE56F130
[11] openttd              0x00007FF7BE751BFE
[12] openttd              0x00007FF7BE4EC405
[13] openttd              0x00007FF7BE4E8B9F
[14] openttd              0x00007FF7BE5014A0
[15] openttd              0x00007FF7BE62998A
[16] openttd              0x00007FF7BE460CF8
[17] openttd              0x00007FF7BE7C7E92
[18] KERNEL32             0x00007FFE1FD47034 BaseThreadInitThunk + 20
[19] ntdll                0x00007FFE211E2651 RtlUserThreadStart + 33

*** End of additional info ***
```

**Comment by James103** [comment]
1. Have you tried reproducing this using the latest nightly? Either download from https://www.openttd.org/downloads/openttd-nightlies/latest.html or switch to the beta channel in Steam.
2. Have you tried setting `video_hw_accel = false` in `%UserProfile%\Documents\OpenTTD\openttd.cfg`?

**Comment by glx22** [comment]
Most likely duplicate of #8967

**Comment by LordAro** [comment]
Duplicate of #8967. Should be fixed in 1.11.2, released now!
